tablo/geom_utils.py

```diff
@@ -437,4 +437,196 @@ def extract_significant_digits(number):
     if is_negative:
         rounded = 0 - rounded
 
-    return rounded
\ No newline at end of file
+    return rounded
+
+
+def wkt_to_esri_feature(wkt):
+    geom_type = wkt[:wkt.find('(')]
+
+    def _geom_repl():
+        bracket_multiplier = 1
+        if geom_type.find('LINESTRING') > -1:
+            bracket_multiplier = 2
+        return json.loads(re.sub(
```

Comment:
It might be better to call `WKT_GEOM_2D_REGEX.sub` instead of `re.sub` with the compiled regex as an argument. I'm assuming both `sub` methods have a similar signatures.